### ISSUE TITLE
Strange bahaviour of suffix in GUI

### ISSUE THREAD

**Issue by HauptlingHD** [k4yt3x/video2x#1263]
Opening a separate issue after #1222 since I have probably discovered a different bug.

In the GUI, the "suffix" option in the "Encoding Options" category is populated with the suffix of the Input file by default, or it should be, I guess.

However, the way this is currently handled leads to issues, if there are multiple dots in a filename.

Example:

File Name:
```
001. Warum betreibt man Astronomie (27.09.98).avi
```
Auto generated Suffix:
```
. Warum betreibt man Astronomie (27.09.98).avi
```

The behavior should be changed, so the suffix is separated by only the last dot in a file. Maybe the program could just search for the dot backwards?

**Comment by k4yt3x** [comment]
Yeah I saw that in a few cases. Looks like I broke the suffix search when I did the overhaul. I'll have it fixed.

**Comment by k4yt3x** [comment]
I fixed it, then decided to disable it and just fix it to .mkv for better compatibility. Will be in the next release.
